dspace-server-webapp/pom.xml

```diff
@@ -241,11 +239,7 @@
         <dependency>
             <groupId>org.springframework.data</groupId>
             <artifactId>spring-data-rest-hal-browser</artifactId>
-            <version>3.1.10.RELEASE</version>
-            <!-- if you get a java.util.zip.ZipException: invalid LOC
-                 header (bad signature) during the tomcat startup
-                 force the use of the previous version as the jar file
-                 looks corrupted in the maven repository -->
+            <version>${spring-hal-browser.version}</version>
```

Comment:
No we cannot drop this, as this dependency `spring-data-rest-hal-browser` is only included in the `dspace-server-webapp`.  I just moved it's version # over to the DSpace Parent POM because it was easier to see/manage alongside the other Spring versions (the last two times I've upgraded Spring, I forgot about this separate dependency in `dspace-server-webapp`)